src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,59 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "Which of the following is corect about a Tree?",
+    Answer: "Tree is minimally connected.",
+    Distractor1: "Tree can be cyclic.",
+    Distractor2: "Tree must be undirected graph",
+    Distractor3: "Tree must have atleast two nodes",
```

Comment:
I think it would sound better to use the plural for trees. 

```suggestion
    Answer: "trees are minimally connected.",
    Distractor1: "trees can be cyclic.",
    Distractor2: "a tree must be an undirected graph",
    Distractor3: "trees must have at least two nodes",
```